tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/SearchQueryService.java

```diff
@@ -121,6 +129,24 @@ private void assertNames(SearchResponse actual, String... expected)
         assertEquals(result, expectedList, "Unexpected search results.");
     }
 
+    private List<String> orderNames(boolean isAscending, String... filename){
+        List<String> orderedNames = Arrays.stream(filename).sorted().collect(Collectors.toList());
+        if(!isAscending){
+            orderedNames = orderedNames.stream().sorted(Comparator.reverseOrder()).collect(Collectors.toList());
+        }
+        return orderedNames;
+    }
```

Comment:
Do we need to sort here ? Why not just pass an expected list (in the expected order) ?

Reply:
Correct me if I'm wrong but I don't think we can do that since the file names have randomly generated prefixes. Thus, we need a method to do the sorting automatically.